Source/DistanceFunctions/arm_chebyshev_distance_f64.c

```diff
@@ -48,27 +48,49 @@
  */
 float64_t arm_chebyshev_distance_f64(const float64_t *pA,const float64_t *pB, uint32_t blockSize)
 {
-   float64_t diff=0.,  maxVal,tmpA, tmpB;
```

Comment:
There should be a test for __aarch64__

I won't add this comment anymore to other files but the problem also exist in other files.